test/WebJobs.Script.Tests/ManagedDependencies/HostFile.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Azure.WebJobs.Script.ManagedDependencies;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests.ManagedDependencies
+{
+    internal class HostFile
+    {
```

Comment:
We already have [HostJsonFileConfigurationSource.cs](https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Config/HostJsonFileConfigurationSource.cs).  Any reason why are defined this?